deploy/olm-catalog/kogito-operator/0.8.0/kogito-operator.v0.8.0.clusterserviceversion.yaml

```diff
@@ -108,9 +108,10 @@ spec:
         name: ""
         version: v1
       specDescriptors:
-      - description: Environment variables used during build time
-        displayName: Build Env Variables
-        path: build.env
+      - description: Environment variables to be added to the runtime container. Keys
```

Comment:
the 0.8.0 version of the CSV should not get updated since we will wait for 0.8.0 to be released to merge this one. Sorry to not tell it before, but the 0.8.0 kept pushing forward. Looks like that now we are about to release and this change will have a huge impact in the tests.

Reply:
Will rebase after the 0.8.0 release.